### PR TITLE
Adds a new achievement and command report for cultists obliterating Centcom's shuttle corps

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
This is based on an occasional gag I like to do while adminning when cultists rapid-fire off 3 shuttle curses one after another, where I send a command report begging whatever's on the station to leave the shuttle workers alone. After all, seeing one case of a coworker mutilate themselves out of nowhere would be disturbing, but having 3 incidents happen almost simultaneously? That'd be a lot to bear!

If a cult uses all 3 of their shuttle curses within 10 seconds of the first one, all members of the cult who are currently connected will get the new achievement, named "WHAT JUST HAPPENED". In addition, a few seconds after the last use, Centcom will send a new report expressing shock and distress at whatever horrible scene just played out on their end.

[](https://i.imgur.com/XmxjuvR.png)

This PR also adds a few new curse announcement messages for good measure.

## Why It's Good For The Game
I think this is a pretty funny niche interaction that could totally catch players off guard for some good laughs.

## Changelog

:cl: Ryll/Shaps
add: Added a new achievement for cultists who like to mess with NT's shuttle corps, "WHAT JUST HAPPENED".
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -546,24 +556,31 @@
...
-			"A shuttle engineer began screaming 'DEATH IS NOT THE END' and ripped out wires until an arc flash seared off her flesh.",
-			"A shuttle inspector started laughing madly over the radio and then threw herself into an engine turbine.",
-			"The shuttle dispatcher was found dead with bloody symbols carved into their flesh.",
-			"The shuttle's transponder is emitting the encoded message 'FEAR THE OLD BLOOD' in lieu of its assigned identification signal.")
-		var/message = pick_n_take(curses)
-		message += " The shuttle will be delayed by three minutes."
-		priority_announce("[message]", "System Failure", 'sound/misc/notice1.ogg')
+
+		var/curse_message = "Something has gone horrendously wrong..."
+		for(var/i in 1 to 3) // can't use pick_and_take() with json, so we just do 3 shots at pulling a message we haven't used yet, probably good enough
```
Comment (Mothblocks) [inline code/modules/antagonists/cult/cult_items.dm:561]: you could make this work but its not really necesasry ig uess

Comment (Mothblocks) [reply code/modules/antagonists/cult/cult_items.dm:561]: you'd just call `json_load(CULT_SHUTTLE_CURSE)`, save its curse_announce field somewhere, then pick'n'take that

File: strings/cult_shuttle_curse.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	],
+
+	"omfg_announce": [
+		"OH MY FUCKING GOD WHAT JUST HAPPENED IN OUR SHUTTLE DOCK",
+		"HOLY SHIT OUR SHUTTLE DOCK IS A WARZONE",
+		"JESUS FUCKING CHRIST LEAVE OUR SHUTTLE WORKERS ALONE",
+		"WHAT THE FUCK",
+		"LEAVE THE SHUTTLE WORKERS ALONE YOU FUCKING ANIMALS",
+		"STOP STOPS STOPST OSPTOPSTOP"
+	]
```
Comment (tralezab) [inline strings/cult_shuttle_curse.json:23]: requesting some form of CHELP

File: code/__DEFINES/misc.dm [hunk 3]
```diff
@@ -415,6 +415,7 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 #define SCAR_LOC_FILE "wounds/scar_loc.json"
 #define EXODRONE_FILE "exodrone.json"
 #define CLOWN_NONSENSE_FILE "clown_nonsense.json"
+#define CULT_SHUTTLE_CURSE "cult_shuttle_curse.json"
```
Comment (tralezab) [inline code/__DEFINES/misc.dm:418]: we should really eventually move either all the string path defines into their own file or files that make the most sense for them in the future

not something you have to do btw just food for thought

File: code/modules/antagonists/cult/cult_items.dm [hunk 4]
```diff
@@ -546,24 +556,31 @@
...
 			to_chat(user, span_danger(span_big("You sense that the emergency escape shuttle can no longer be cursed. It would be unwise to create more cursed orbs.")))
 		else if(MAX_SHUTTLE_CURSES-totalcurses == 1)
 			to_chat(user, span_danger(span_big("You sense that the emergency escape shuttle can only be cursed one more time.")))
 		else
 			to_chat(user, span_danger(span_big("You sense that the emergency escape shuttle can only be cursed [MAX_SHUTTLE_CURSES-totalcurses] more times.")))
+
+		if(totalcurses >= MAX_SHUTTLE_CURSES && (world.time < first_curse_time + SHUTTLE_CURSE_OMFG_TIMESPAN))
+			var/omfg_message = pick_list(CULT_SHUTTLE_CURSE, "omfg_announce") || "LEAVE US ALONE!"
+			addtimer(CALLBACK(GLOBAL_PROC, .proc/priority_announce, omfg_message, "Priority Alert", 'sound/misc/notice1.ogg'), rand(2 SECONDS, 6 SECONDS))
+			for(var/mob/iter_player in GLOB.player_list)
```
Comment (tralezab) [inline code/modules/antagonists/cult/cult_items.dm:580]:
```suggestion
			for(var/mob/iter_player as anything in GLOB.player_list)
```
everything in the player list is a mob so should be fine

**Review by tralezab — COMMENTED** [review]
beep boop good pr son

File: strings/cult_shuttle_curse.json (hunk 2 above)
Comment (Mothblocks) [reply strings/cult_shuttle_curse.json:23]: ;;;;;;;HELP

**Review by Mothblocks — CHANGES_REQUESTED** [review]
merge from upstream so ci can pass

**Pull request merged**